src/databricks/labs/ucx/source_code/base.py

```diff
@@ -54,6 +56,15 @@ def as_deprecation(self) -> 'Deprecation':
     def as_convention(self) -> 'Convention':
         return Convention(**self.__dict__)
 
+    def for_path(self, path: Path) -> LocatedAdvice:
+        return LocatedAdvice(path=path, advice=self)
```

Comment:
```suggestion
        return LocatedAdvice(self, path)
```
there's not much point in named arguments here.

Reply:
done